README.md

```diff
@@ -316,61 +316,26 @@ Getting the last authored Item, passing the request from standard input.
 	]
 
 Documentation about all the options is available with the `--help` flag to both
-the base tool, and each API. (Don't trust this example, run those for real to
-get an up-to-date help).
+the base tool, and each API.
 
 	$ lrn-cli --help
-	Usage: lrn-cli [OPTIONS] COMMAND [ARGS]...
```

Comment:
I removed this as I was relying on it and totally missed the "Don't trust this example" bit and I'm sure others will too =D